news/2022-10-04-changes-to-osu-mania-sr-and-pp.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Changes to osu!mania Star Rating & Performance Points"
+date: 2022-10-04 12:00:00 +0000
+---
+
+Big changes are arriving in the osu!mania scene as difficulty calculation receives its first set of updates in a long time! Read on for more info!
+
+![](/wiki/shared/news/banners/star-rating.jpg)
+
+As these new changes are the first to be made in nearly [4 years](https://osu.ppy.sh/home/news/2018-05-16-performance-updates) for performance points and a whopping 8 years for star rating, it may be helpful to check the respective wiki pages for [star rating](/wiki/Beatmap/Star_rating) and [performance points](/wiki/Performance_points) to get a quick run-down on how everything works.
+
+Let's dive straight into what's new:
+
+## osu!mania changes
+
+### Long note difficulty changes
+
+A [new change](https://github.com/ppy/osu/pull/17913) in long note calculation fixes issues with fast releases giving disproportionately more star rating than they should.
+
+Before this change, all overlapping long notes would receive a fixed bonus. After this change, this bonus now scales proportionally with how long ago the last long note release was.
+
+This change also fixes an oversight with long notes in the rightmost column of a map taking precedence in overlap calculation.
+
+An [interactive demonstration](https://www.desmos.com/calculator/7eboanfzne) by [**Evening**](https://osu.ppy.sh/users/2193881) helps visualize how this change affects the long note bonus if you're interested.
+
+### Chord difficulty changes
+
+An [issue has been fixed](https://github.com/ppy/osu/pull/19120) where star rating would change depending on the order of notes in the beatmap file.
+
+This change makes sure that the difficulty of the entire chord is taken into account, rather than only the last note to be processed by the difficulty calculator.
+
+This primarily affects maps with heavy usage of chords (especially in higher key modes), for example:
+
+- [Cardboard Box - The Limit Does Not Exist [[7K] Infinity]](https://osu.ppy.sh/beatmapsets/379758#mania/831204): 7.45\* --> 7.71\*
+- [Manabu Namiki - Tenshi (True Final Boss) [[4K] MIPHATHER]](https://osu.ppy.sh/beatmapsets/1503789#mania/3080978): 5.36\* --> 5.44\*
+
+### Using accuracy instead of score for performance points
+
+Overall score has historically been used as the measure of skill a player demonstrates on a given map. With the new direction [osu!(lazer)](https://github.com/ppy/osu) is taking, it no longer makes sense to base performance entirely on this metric.
+
+With a [new change](https://github.com/ppy/osu/pull/18749), awarded pp will now be based on the accuracy the player gets on the map using this formula:
+
+put accuracy formula image here
```

Comment:
still missing an image for this

Reply:
Would this be adequate (in terms of formatting)?

![Figure_1](https://user-images.githubusercontent.com/43741444/194537752-a60eb9e8-658e-426b-9fe1-8a9bb859b7e1.png)

I used the matplotlib style template, but I can't seem to get the right font to be used. If someone wants to reproduce it with the proper font (Inter) or make tweaks, I attach a gist to the script to generate these:

https://gist.github.com/Kominaru/931e5e44ec983d8ce5a87a447f13a6d0

Reply:
Remade image with correct font